mclagsyncd/mclaglink.h

```diff
@@ -35,23 +35,67 @@
 #include <set>
 
 #include "producerstatetable.h"
+#include "subscriberstatetable.h"
+#include "select.h"
 #include "selectable.h"
 #include "mclagsyncd/mclag.h"
+#include "notificationconsumer.h"
+#include "notificationproducer.h"
 
-namespace swss {
+#define ETHER_ADDR_LEN 6
```

Comment:
I think this should be already defined in `<net/ethernet.h>`

Reply:
addressed it